include/blosc2.h

```diff
@@ -1909,6 +1909,14 @@ BLOSC_EXPORT int blosc2_vlmeta_get(blosc2_schunk *schunk, const char *name,
  */
 BLOSC_EXPORT int blosc2_vlmeta_delete(blosc2_schunk *schunk, const char *name);
 
+/**
+ * @brief Get a list of all the variable-length metalayer names
+ * @param schunk The super-chunk containing the variable-length metalayers.
+ * @param names The pointer to a char** to store the name pointers.
+ * @return If successful, the number of the variable-length metalayers in the super-chunk.
```

Comment:
Add an empty line before the `@return` docstring.

Reply:
Also, it would be convenient to to add a note saying that the user should `free(*names)` to avoid leaks.

Reply:
Fixed in 52736067b453102049a645aa7137388b5a23e4be.